controllers/submariner/submariner_controller_test.go

```diff
@@ -239,14 +238,16 @@ func testDeletion() {
 
 		t.initClientObjs = append(t.initClientObjs,
 			t.newDaemonSet(names.GatewayComponent),
-			t.newPodWithLabel("app", names.GatewayComponent))
+			t.newPodWithLabel("app", names.GatewayComponent),
+			t.newDaemonSet(names.RouteAgentComponent))
 	})
 
-	It("should uninstall the component resources and remove the finalizer", func() {
+	FIt("should uninstall the component resources and remove the finalizer", func() {
```

Comment:
Typo? s/FIt/It

Reply:
F = focus - I removed the F in the next commit.

Reply:
Ah okay, thanks/learning :nerd_face: :+1:

Reply:
(I noticed it went away btw, that's part of why I thought it was a typo)